### ISSUE TITLE
utils modules

### ISSUE THREAD

**Issue by djguiaume** [djguiaume/AcsilServer2#1]
I was thinking about using those : 
### Module

OAuth2
https://github.com/thomseddon/node-oauth2-server
ORM
https://github.com/dresende/node-orm2
Routing
http://expressjs.com/guide/routing.html
acl
https://github.com/OptimalBits/node_acl
### Build/packaging

https://github.com/jaredallard/nexe
https://travis-ci.org/
### enhencement

server side event (SSE)
http://www.html5rocks.com/en/tutorials/eventsource/basics/?redirect_from_locale=fr
chunk upload ?
http://www.componentix.com/blog/9/file-uploads-using-nodejs-now-for-real

What do you think ? @Delmael @eohland @ngfx @fguf

**Comment by eohland** [comment]
Yeah, looks good to me. Express is well known but maybe not the best choice. What do you think about [hapi](http://hapijs.com/) for the HTTP / Routing / Auth?
